oat-sa/tao-item-runner-qti-fe
# UNO-235 move ckeditor, simulator to tao-core-libs-fe

Related to: https://oat-sa.atlassian.net/browse/UNO-235

`CKeditor` and `simulator` were moved from `tao-core`, `tao-core-ui-fe`,` tao-item-runner-qti-fe` to `tao-core-shared-libs-fe` to reduce duplication of code https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1
How to update CKeditor in `lib/ckeditor/README.md` in https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1

How to test:
`npm run test`

**package.json should be updated after merge tao-core-shared-libs-fe:**

- [x] https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1

Related PRs:
https://github.com/oat-sa/tao-core-ui-fe/pull/123
https://github.com/oat-sa/extension-tao-devtools/pull/178
https://github.com/oat-sa/tao-core/pull/2498
